bitnami/airflow/README.md

```diff
@@ -144,13 +144,15 @@ The following tables lists the configurable parameters of the Airflow chart and
 | `postgresql.postgresqlUsername`           | Airflow Postgresql username                                                                          | `bn_airflow`                                                 |
 | `postgresql.postgresqlPassword`           | Airflow Postgresql password                                                                          | `nil`                                                        |
 | `postgresql.postgresqlDatabase`           | Airflow Postgresql database                                                                          | `bitnami_airflow`                                            |
+| `postgresql.existingSecret`         | Name of an existing secret containing the PostgreSQL password ('postgresql-password' key) . This secret is used in case of postgresql.enabled=true and we would like to specify password for newly created postgresql instance           | `nil`
```

Comment:
Could you fix the allignment of the new entry in the values table, so it looks nicer in text editors? Please also check "redis.existingSecret"

Reply:
Thanks for the review. Alignment adjusted